docs/src/adoc/index.adoc

```diff
@@ -597,6 +597,44 @@ LocalDate releaseDate = handle.createQuery(
     .findOnly();
 ----
 
+==== LIKE clauses
+
+Due to a built-in safety mechanism, `LIKE` clauses and argument binding
+don't work together intuitively and must be written a particular way. The following
```

Comment:
I somewhat disagree with the wording here.  This is very intuitive behavior if you understand how the driver works -- bound parameters must be proper first class parameters to the query, not some templated string.  It'd be extremely surprising for it to template *into* string literals.

I'm going to do a followup here to reword this since I didn't get a chance to review this in the first place.

Reply:
> if you understand how the driver works

:)

But fair enough, manuals are there to be amended